### DISCUSSION TITLE
Easier installing Windows Terminal in Windows Server 2022

### DISCUSSION THREAD

**Discussion by davetapley** [Textualize/textual#4484] [Show and tell]
Textual apps look pretty terrible in stock command prompt / Powershell on Windows Server 2022.

And for reasons (https://github.com/microsoft/winget-cli/discussions/2361) installing the new Windows Terminal requires quite a few manual steps.

To simplify this I just released this which is an installer `.exe`:
https://github.com/JEFuller/install-windows-terminal/releases/tag/v1.0.0

Feedback welcome!

**Comment by davep** [comment]
What advantage does this have over [the recommended installation method of simply installing from the Windows Store](https://github.com/microsoft/terminal?tab=readme-ov-file#microsoft-store-recommended)? (asking as someone who tries not to use Windows if he can help it)

**Reply by davetapley** [reply]
Windows Store isn't supported on their [Windows Server](https://en.wikipedia.org/wiki/Windows_Server) products,
which I learned only after suggesting the same thing to a client 😞

**Reply by davep** [reply]
Oh, ouch, thanks for documenting that.
